src/Logs/Providers/DefaultLoggingProvider.cs

```diff
@@ -63,5 +70,20 @@ public static Serilog.ILogger CreateBootstrappLogger(string applicationName)
                 .Enrich.WithProperty("Environment", Environment.GetEnvironmentVariable("ASPNETCORE_ENVIRONMENT") ?? "Local")
                 .Enrich.WithProperty("ApplicationVersion", Environment.GetEnvironmentVariable("APPLICATION_VERSION") ?? "v0.0.0");
         }
+
+        private static LoggerConfiguration BaseConfiguration(this LoggerConfiguration loggerConfiguration,
+            IServiceProvider services, string applicationName)
+        {
+            return (Environment.GetEnvironmentVariable("PROTEUS__DEFAULT_LOG_LEVEL") switch
```

Comment:
same here (var name would be DEFAULT_LOG_LEVEL)

Reply:
Fixed